.github/workflows/reusable_receptorctl_checks.yml

```
@@ -0,0 +1,34 @@
+---
+name: lint-receptorctl
+
+"on":
+  workflow_call:
+
+jobs:
+  lint-receptorctl:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+        - 3.11
+        session:
+        - check_style
+        - check_format
+
+    name: "Run receptorctl ${{ matrix.session }} session"
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Setup nox
+      uses: wntrblm/nox@2024.04.15
+      with:
+        python-versions: "${{ matrix.python-version }}"
+
+    - name: "Run receptorctl nox ${{ matrix.session }} session"
+      run: |
+        nox -s "${{ matrix.session }}"
+      working-directory: ./receptorctl
```

Let's reshuffle this for consistency with the other workflow:
```suggestion
    env:
      NOXSESSION: ${{ matrix.session }}

    name: >- # can't use `env` in this context:
      Run `receptorctl` ${{ matrix.session }} session

    steps:
    - name: Set up nox
      uses: wntrblm/nox@2024.04.15
      with:
        python-versions: ${{ matrix.python-version }}

    - name: Check out the source code from Git
      uses: actions/checkout@v4
      with:
        fetch-depth: 0
        sparse-checkout: receptorctl

    - name: >-
        Run `receptorctl` ${{ env.NOXSESSION }} session
      run: |
        nox
      working-directory: ./receptorctl
```